dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -219,7 +219,7 @@ public Response getType(@PathParam("idOrVar") final String idOrVar, @Context fin
 		ContentTypeAPI tapi = APILocator.getContentTypeAPI(user, true);
 		Response response = Response.status(404).build();
 		try {
-			ContentType type = tapi.find(idOrVar);
+			ContentType type = tapi.find(idOrVar); // todo: include here the findSchemesForStruct
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)